data/prod/scenario-25.json

```diff
@@ -142,8 +142,8 @@
                 "model": {
                   "width": 640,
                   "height": 420,
```

Comment:
The image ratio here is not 16:9, whereas the image itself is 16:9. 
As soon as we use the height value as well as the width value in our code, this mismatch between the ratio in the data and that of the original image will make images on our Test/Live scenario pages look skewed. 
Because of this, could you please update all the landscape images to have a 16:9 ratio in the data, such as :
```
"width": 1024,
"height": 576,
```